tests/common/gen.bash

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+declare tests root
+tests="$(dirname "$(dirname "$(readlink -f "$0")")")"
+root="$(dirname "${tests}")"
+
+cypress_gen() {
+  local file
+
+  file="$(readlink -f "$1")"
+
+  local -a input
+
+  readarray -t input < "${file}"
+
+  local describe test
+
+  local -a its
```

Comment:
Might be a dumb question but what is the acronym here? :sweat_smile:

Reply:
It collects the `it`s declared in the `.cy.js` files, so it is not really an acronym :sweat_smile:

Reply:
Aaah now it all makes sense, thanks :+1: